src/app/static/src/app/components/dataExploration/DataExploration.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="content">
+        <div class="pt-4">
+            <adr-integration></adr-integration>
+            <upload-inputs v-if="step === 1"></upload-inputs>
+            <review-inputs v-if="step === 2"></review-inputs>
+            <button type="button"
+                    class="btn btn-white shadow-none"
+                    v-translate="'back'"
+                    @click="back">
+            </button>
+            <button type="button"
+                    class="btn btn-red shadow-none"
+                    v-translate="'next'"
+                    @click="next">
+            </button>
```

Comment:
I guess we should also disable back if step === 1 and disable next if step === 2